src/NexusMods.App.UI/Pages/LoadoutPage/LoadoutViewModel.cs

```diff
@@ -100,6 +100,7 @@ public LoadoutViewModel(IWindowManager windowManager, IServiceProvider servicePr
 
         ViewFilesCommand = viewModFilesArgumentsSubject
             .Select(viewModFilesArguments => viewModFilesArguments.HasValue)
+            .ObserveOnUIThreadDispatcher()
```

Comment:
Already on UI, this can be removed.

Reply:
Subject seems to actually be signaled from thread pool, line 179

Reply:
Yes, because you removed `ObserveOnUIThreadDispatcher` from line 177.